src/Introduction.mdx

```diff
@@ -0,0 +1,46 @@
+# Front End Interview — Design System
+
+## Hi and welcome to our code challenge. 👋
+
+Make sure to carefully read the following brief, containing all the details you need.
+
+## The Goal
+
+For this take-home test, the goal is to develop a Tabs component that will be part of our Design System project.
+The component should be accessible and reusable, following best practices in front-end development.
+
+## Acceptance Criteria
+
+Share with us the 🔗 link to your repository (don’t fork please) that contains everything you need to show us about the component and its features.
+More exactly, we write some user stories:
+
+- As a developer using the Tabs, I want to be able to switch between different variants of the tabs, according to the design.
+- As a developer using the Tabs, I want some Tab to show additional information with a Badge, and I want to choose also an appropriate Badge variant.
```

Comment:
- As a developer (Api end user), I can add a badge to the Tab to show additional information. 

- As a developer (Api end user), I can choose between different badge variants to add to the Tab.